tools/wptrunner/wptrunner/browsers/base.py

```diff
@@ -365,10 +374,10 @@ def _run_server(self, group_metadata, **kwargs):
             self._output_handler.start(group_metadata=group_metadata, **kwargs)
         self.logger.debug("_run complete")
 
-    def stop(self, force=False):
+    def stop(self, force: bool = False) -> bool:
         self.logger.debug("Stopping WebDriver")
         clean = True
-        if self.is_alive():
+        if self._proc and self.is_alive():
```

Comment:
We seem to end up checking for `self._proc` twice, I think we could remove this one.